test/doctests.hs

```diff
@@ -1,4 +1,4 @@
 import Test.DocTest
 
 main :: IO ()
-main = doctest ["-isrc", "-XOverloadedStrings", "src/Utils.hs"]
+main = doctest ["-isrc", "-XOverloadedStrings", "src/Version.hs", "src/GH.hs"]
```

Comment:
It would be nice to make the file list dynamic.

Reply:
Hmm, apparently, it should be possible to just list `src/` https://hackage.haskell.org/package/doctest-0.16.0.1/docs/Test-DocTest.html#v:doctest

Reply:
But it seems very slow, even with the `--fast` argument.